stackslib/src/net/prune.rs

```diff
@@ -71,10 +71,12 @@ impl PeerNetwork {
                         Some(peer) => {
                             let stats = convo.stats.clone();
                             let org = peer.org;
-                            if org_neighbor.contains_key(&org) {
-                                org_neighbor.get_mut(&org).unwrap().push((nk, stats));
+                            if let std::collections::hash_map::Entry::Vacant(e) =
```

Comment:
I really don't like the `hash_map::Entry` API.  Can you instead factor this as follows:

```rust
if let Some(stats_list) = org_neighbor.get_mut(&org) {
   stats_list.push((nk, stats));
} else {
   org_neighbor.insert(org, vec![(nk, stats)]);
}
```

Reply:
Sure